keyboards/handwired/derb_planck/rev1/config.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 Bernd Lehmann (@der-b)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/*
+ * Feature disable options
+ *  These options are also useful to firmware size reduction.
+ */
+
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+//
+#define QMK_ESC_OUTPUT D0 // COL pin if COL2ROW
+#define QMK_ESC_INPUT B1  // ROW pin if COL2ROW
+
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 0
+
+#define NO_USB_STARTUP_CHECK
+
```

Comment:
In this case I only removed the first two defines. NO_USB_STARTUP_CHECK is essential for the functionality of the keyboard. The ATmega32u4 is not directly connected to a USB port. It uses an FSUSB36 to multiplex between three USB ports. There are shortcuts to change the active port. There are two issues, if NO_USB_STARTUP_CHECK is not defined:

- If only one USB cable is inserted to an inactive USB port, the keyboard has power, but the there is still no USB connection since the data lines are not connected to the ATmega32u4. Without this define, it is not possible to use a shortcut to activate this port, since the matrix scanning is deactivated. 
- If one accidentally switch from an active USB port to an inactive one, than the firmware disables the matrix scanning and it is not possible to switch back by shortcut anymore. 

The second issue could be circumvented by a proper hardware design, which detects active USB ports before switching. Unfortunately, while designing the PCB, I didn't realized, that the ATmage32u4 can not configure a pull-down resistor on GPIOs. I'm used to work with ARM MCUs, which have configurable pull-down resistors. Therefore, the detection lines are useless in the current version of the board. But then still there is the first issue.

Reply:
Yes sorry, I meant for this to be migrated to `info.json` as well. See sigprof's comments below.